site/app/views/submission/HomeworkView.php

```diff
@@ -726,15 +726,25 @@ function submitStudentGradeable(student_id, highest_version) {
 HTML;
 
                                     $title = "";
+                                    $visible1 = "visible";
                                     $return .= <<<HTML
                             <div class='diff-element'>
 HTML;
                                     if ($diff_viewer->hasDisplayExpected()) {
                                         $title = "Student ";
                                     }
+                                    if ($diff_viewer->hasDisplayActual()) {
+                                        //I unfornately have no idea how the strlen is measured. It's not the number of characters in there
+                                        //an extra 250ish characters are added on for some reason.
+                                        if (strlen($diff_viewer->getDisplayActual()) < 350) {
```

Comment:
We need to investigate this further before accepting this PR as this is subject to change/break without anyone realizing it with this "magic number". Use var_dump to see what the content is? I'm guessing there's weird space characters in the text that are messing things up if using `len` in Python on the file contents also says 350 characters.